addons/base_automation/static/src/kanban_header_patch.js

```diff
@@ -0,0 +1,73 @@
+/* @odoo-module */
+
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+import { patch } from "@web/core/utils/patch";
+import { KanbanHeader } from "@web/views/kanban/kanban_header";
+import { TRIGGER_FILTERS } from "./utils";
+
+const SUPPORTED_TRIGGERS = [
+    "on_stage_set",
+    "on_tag_set",
+    "on_state_set",
+    "on_priority_set",
+    "on_user_set",
+    "on_archive",
+    "on_unarchive",
+];
```

Comment:
@bouvyd can you have a look at this ?

From a grouped kanban, open automations.
If it was grouped by a field that is compatible with a trigger from this list, it will become part of the context (default_trigger).

There is a problem here though, 'on_archive' & 'on_unarchive' are based on the same field. 'on_archive' will won over 'on_unarchive' (see enrichContext function below).

Maybe we would like to take into account the group value in order to pick the proper trigger in that case ?

Reply:
not that's unnecessary, current behaviour is fine

users still have plenty to confirm when doing these flows, we're just using sensible defaults. archival is more frequent than unarchival in any case (nearly by definition...)